src/core/stdcpp/string.d

```diff
@@ -328,6 +328,121 @@ extern(D):
 
         _String_alloc!(_String_base_types!(T, Alloc)) _Base;
     }
+    else version (CppRuntime_Gcc)
+    {
+        ///
+        this(DefaultConstruct)                                              { _M_p = _M_local_data(); _M_set_length(0); }
```

Comment:
If _you_ don't need to support it, I'd say defer adding the `CppRuntime_Gcc` implementation. Note that because there is no instantiations of the template in druntime, you can probably just `static assert(0, "CppRuntime_Gcc not yet supported");`

 If you do... you'll need to wait until DIP1014 is implemented.

Reply:
I tickled my own needs months ago... I don't need to do all this work to make the tests work and hit all platforms ;)
I just want to put the lid on this whole thing... it's gone on way too long.

Reply:
This is a major problem. AFAIK nobody is working on DIP 1014.

Reply:
Well, it's sitting on your review queue... is there something wrong with it? Are you planning on rejecting it?
It's literally the thing on your desk right now isn't it? (according to the DIP schedule)

Reply:
I assume he means implementation, but it is ambiguous.

Reply:
DIP 1014 looks simple. I'm sure it'll appear promptly if it's approved.

Reply:
I meant internal pointers in std::string are a major problem, and that nobody is working on DIP 1014.

Reply:
Right, but DIP 1014 is resting with you and Walter is it not? You are the only one that can move that along.

Reply:
@TurkeyMan DIP 1014 have been recently approved